x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_stop/stop_button_modal.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiConfirmModal, EuiOverlayMask, EUI_MODAL_CONFIRM_BUTTON } from '@elastic/eui';
+
+import { ForceStopAction } from './use_force_stop_action';
+
+export const StopButtonModal: FC<ForceStopAction> = ({
+  closeModal,
+  item,
+  forceStopAndCloseModal,
+}) => {
+  return (
+    <>
+      {item !== undefined && (
+        <EuiOverlayMask>
+          <EuiConfirmModal
+            title={i18n.translate('xpack.ml.dataframe.analyticsList.forceStopModalTitle', {
+              defaultMessage: 'Force stop {analyticsId}',
```

Comment:
In case the job name is long, can we put it in the body of the message instead of the title?
```suggestion
              defaultMessage: 'Force this job to stop?',
```